src/components/MAPI/keypairs/permissions/usePermissionsForKeyPairs.ts

```diff
@@ -0,0 +1,33 @@
+import { Providers } from 'model/constants';
+
+import { usePermissionsForCertConfigs } from './usePermissionsForCertConfigs';
+import { usePermissionsForStorageConfigs } from './usePermissionsForStorageConfigs';
+
+export function usePermissionsForKeyPairs(
+  provider: PropertiesOf<typeof Providers>,
+  namespace: string
+) {
+  const { canCreate: canCreateCertConfigs } = usePermissionsForCertConfigs(
+    provider,
+    namespace
+  );
+
+  const { canGet: canGetStorageConfigs } = usePermissionsForStorageConfigs(
```

Comment:
As the client certificates table and the creation guide reference different resources, we determine permissions separately for fetching (can `get` `StorageConfigs`) and for creation (can `create` `CertConfigs`)